compiler/luci-interpreter/src/kernels/CMakeLists.txt

```diff
@@ -1,234 +1,60 @@
-find_package(Threads REQUIRED)
+if (NOT LUCI_MICRO)
+  find_package(Threads REQUIRED)
```

Comment:
I think this line is redundant, this package should be found during `initialize_pal` call